test/manual/definitions/apm/.net/linux/ubuntu20-aspnetcore.json

```diff
@@ -0,0 +1,50 @@
+{
+    "global_tags": {
+        "owning_team": "OpenSource",
+        "Environment": "development",
+        "Department": "Product",
+        "Product": "Virtuoso"
+    },
+
+    "resources": [{
+        "id": "ubuntunetcore1",
+        "display_name": "NetCoreUbuntu20Host",
+        "provider": "aws",
+        "type": "ec2",
+        "size": "t3.nano",
```

Comment:
I'm surprised we can fit so much on a nano, could you double check newrelic to make sure we're not in the 80%+ range for memory usage?

Reply:
Looks like it's using less than half of the memory:

```
ubuntu@ip-172-31-16-179:~$ free
              total        used        free      shared  buff/cache   available
Mem:         467076      216548       20560         788      229968      225988
Swap:             0           0           0
```